### PR TITLE
#484 measure problems on gingerbread

### LINKED ISSUE #807: ListPreference not working on some devices
When I run HoloEverywhere Demo on my Sony Xperia S, go to Settings, click on "List" preference, the radio button on the list is not showed. Moreover, when I try to scroll the list, the app was crashed
Here is the error stack:

03-17 23:58:07.133: E/AndroidRuntime(13717): FATAL EXCEPTION: main
03-17 23:58:07.133: E/AndroidRuntime(13717): java.lang.IllegalArgumentException: bitmap size exceeds 32bits
03-17 23:58:07.133: E/AndroidRuntime(13717):    at android.graphics.Bitmap.nativeCreate(Native Method)
03-17 23:58:07.133: E/AndroidRuntime(13717):    at android.graphics.Bitmap.createBitmap(Bitmap.java:477)
03-17 23:58:07.133: E/AndroidRuntime(13717):    at android.view.View.buildDrawingCache(View.java:6683)
03-17 23:58:07.133: E/AndroidRuntime(13717):    at android.view.View.getDrawingCache(View.java:6534)
03-17 23:58:07.133: E/AndroidRuntime(13717):    at android.view.ViewGroup.drawChild(ViewGroup.java:1664)
03-17 23:58:07.133: E/AndroidRuntime(13717):    at android.view.ViewGroup.dispatchDraw(ViewGroup.java:1466)
03-17 23:58:07.133: E/AndroidRuntime(13717):    at android.widget.AbsListView.dispatchDraw(AbsListView.java:1570)
03-17 23:58:07.133: E/AndroidRuntime(13717):    at android.widget.ListView.dispatchDraw(ListView.java:3139)
03-17 23:58:07.133: E/AndroidRuntime(13717):    at android.view.View.draw(View.java:7093)
03-17 23:58:07.133: E/AndroidRuntime(13717):    at android.widget.AbsListView.draw(AbsListView.java:2699)
03-17 23:58:07.133: E/AndroidRuntime(13717):    at org.holoeverywhere.widget.ListView.draw(ListView.java:192)
03-17 23:58:07.133: E/AndroidRuntime(13717):    at android.view.ViewGroup.drawChild(ViewGroup.java:1739)
03-17 23:58:07.133: E/AndroidRuntime(13717):    at android.view.ViewGroup.dispatchDraw(ViewGroup.java:1466)
03-17 23:58:07.133: E/AndroidRuntime(13717):    at android.view.View.draw(View.java:6989)
03-17 23:58:07.133: E/AndroidRuntime(13717):    at android.view.ViewGroup.drawChild(ViewGroup.java:1739)
03-17 23:58:07.133: E/AndroidRuntime(13717):    at android.view.ViewGroup.dispatchDraw(ViewGroup.java:1466)
03-17 23:58:07.133: E/AndroidRuntime(13717):    at android.view.View.draw(View.java:6989)
03-17 23:58:07.133: E/AndroidRuntime(13717):    at android.view.ViewGroup.drawChild(ViewGroup.java:1739)
03-17 23:58:07.133: E/AndroidRuntime(13717):    at android.view.ViewGroup.dispatchDraw(ViewGroup.java:1466)
03-17 23:58:07.133: E/AndroidRuntime(13717):    at android.view.ViewGroup.drawChild(ViewGroup.java:1737)
03-17 23:58:07.133: E/AndroidRuntime(13717):    at android.view.ViewGroup.dispatchDraw(ViewGroup.java:1466)
03-17 23:58:07.133: E/AndroidRuntime(13717):    at android.view.ViewGroup.drawChild(ViewGroup.java:1737)
03-17 23:58:07.133: E/AndroidRuntime(13717):    at android.view.ViewGroup.dispatchDraw(ViewGroup.java:1466)
03-17 23:58:07.133: E/AndroidRuntime(13717):    at android.view.View.draw(View.java:6989)
03-17 23:58:07.133: E/AndroidRuntime(13717):    at android.widget.FrameLayout.draw(FrameLayout.java:361)
03-17 23:58:07.133: E/AndroidRuntime(13717):    at com.android.internal.policy.impl.PhoneWindow$DecorView.draw(PhoneWindow.java:1862)
03-17 23:58:07.133: E/AndroidRuntime(13717):    at android.view.ViewRoot.draw(ViewRoot.java:1546)
03-17 23:58:07.133: E/AndroidRuntime(13717):    at android.view.ViewRoot.performTraversals(ViewRoot.java:1282)
03-17 23:58:07.133: E/AndroidRuntime(13717):    at android.view.ViewRoot.handleMessage(ViewRoot.java:1884)
03-17 23:58:07.133: E/AndroidRuntime(13717):    at android.os.Handler.dispatchMessage(Handler.java:99)
03-17 23:58:07.133: E/AndroidRuntime(13717):    at android.os.Looper.loop(Looper.java:138)
03-17 23:58:07.133: E/AndroidRuntime(13717):    at android.app.ActivityThread.main(ActivityThread.java:3701)
03-17 23:58:07.133: E/AndroidRuntime(13717):    at java.lang.reflect.Method.invokeNative(Native Method)
03-17 23:58:07.133: E/AndroidRuntime(13717):    at java.lang.reflect.Method.invoke(Method.java:507)
03-17 23:58:07.133: E/AndroidRuntime(13717):    at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:878)
03-17 23:58:07.133: E/AndroidRuntime(13717):    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:636)
03-17 23:58:07.133: E/AndroidRuntime(13717):    at dalvik.system.NativeStart.main(Native Method)

Please help to fix this issue.

Thank you

### PR DESCRIPTION
**Pull request by lgawin**
Here are some changes that fix problems discovered on Sony Xperia with Gingerbread and other GB devices in some circumstances.

Don't hesitate to contact me for details.

I guess it also fixes #807.

### CODE REVIEW

File: library/res/values/rb_styles.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources>
+<resources xmlns:tools="http://schemas.android.com/tools">
```
Comment (Prototik) [inline library/res/values/rb_styles.xml]: But resbuilder will remove this xmlns declaration with the first trying to package library. Please remove it and it's usages below, then I'll merge this pr.

File: library/src/org/holoeverywhere/widget/LinearLayout.java [hunk 2]
```diff
@@ -701,9 +703,9 @@ void measureHorizontal(int widthMeasureSpec, int heightMeasureSpec) {
         }
         maxHeight += getPaddingTop() + getPaddingBottom();
         maxHeight = Math.max(maxHeight, getSuggestedMinimumHeight());
-        setMeasuredDimension(widthSizeAndState | childState & MEASURED_STATE_MASK,
-                ViewCompat.resolveSizeAndState(maxHeight, heightMeasureSpec,
-                        childState << MEASURED_HEIGHT_STATE_SHIFT));
+        setMeasuredDimensionsCompat(widthSizeAndState | childState & MEASURED_STATE_MASK,
```
Comment (Prototik) [inline library/src/org/holoeverywhere/widget/LinearLayout.java]: `setMeasuredDimensionCompat`, not `setMeasuredDimensionsCompat`

**Comment by kkocel** [comment]
It also relates to https://github.com/Prototik/HoloEverywhere/issues/484

File: library/res/values/rb_styles.xml (hunk 1 above)
Comment (lgawin) [inline library/res/values/rb_styles.xml]: Ok, as I wrote I never used resbuilder. So please check also the change in yml file. 
How can I invoke resbuilder?

File: library/src/org/holoeverywhere/widget/LinearLayout.java (hunk 2 above)
Comment (lgawin) [inline library/src/org/holoeverywhere/widget/LinearLayout.java]: Good catch, I'll correct that

File: library/res/values/rb_styles.xml (hunk 1 above)
Comment (Prototik) [inline library/res/values/rb_styles.xml]: Any gradle-based action. Eg. compiling demo project or something else.
But you can invoke it manually: ./gradlew resbuilderMain (or run this task in studio's menu)

**Comment by kkocel** [comment]
@Prototik Seems that build tools are mising...

**Pull request merged**